addons/sale_timesheet/models/project_project.py

```diff
@@ -519,3 +519,20 @@ def _get_profitability_items(self, with_action=True):
             super()._get_profitability_items(with_action),
             with_action
         )
+
+    def _is_project_linked(self):
+        res = super()._is_project_linked()
+        timesheet_linked_count = self.env['account.analytic.line'].search_count([('project_id', '=', self.id)], limit=1)
+        if timesheet_linked_count:
+            timesheet_linked_count = [self.env._("This project is current linked to timesheet.")]
+            if res:
+                res += timesheet_linked_count
+            else:
+                res = timesheet_linked_count
+        return res
```

Comment:
What should we do in that case? It is good to send a message but if we do nothing during the conversion then the message is a bit useless, IMO.